src/routes/transactions/transactions-history.controller.spec.ts

```diff
@@ -355,30 +399,89 @@ describe('Transactions History Controller (Unit)', () => {
       .get(`/v1/chains/${chainId}/safes/${safeAddress}/transactions/history/`)
       .expect(200)
       .then(({ body }) => {
-        expect(body.results).toHaveLength(
-          6, //3 date labels and 3 transactions
-        );
-        expect(body.results[1]).toEqual(
-          getJsonResource('module-transactions/expected-response.json')
-            .results[0],
-        );
-        expect(body.results[3]).toEqual(
-          getJsonResource('multisig-transactions/erc20/expected-response.json')
-            .results[0],
-        );
-        expect(body.results[5]).toEqual(
-          getJsonResource(
-            'incoming-transfers/native-coin/expected-response.json',
-          ).results[0],
-        );
+        expect(body.results).toHaveLength(6); //3 date labels and 3 transactions
```

Comment:
Nit: let's also add expectations for the date labels to be thorough.

Reply:
Good point! 😄 Added in [acc2269](https://github.com/safe-global/safe-client-gateway-nest/pull/503/commits/acc2269e988c9ad8a4d68e8731715cf95578a824)

Reply:
I had to relax the match conditions for labels timestamps in [b45627c](https://github.com/safe-global/safe-client-gateway-nest/pull/503/commits/b45627cbd4fab5caffb4dc703b04a632035cc2a4), because these labels are set to the execution day start, and the epoch for the day start depends on the time zone of the machine executing the tests.